tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ServiceCommand.java

```diff
@@ -81,7 +82,6 @@
         "com.liferay.portal.service.http",
         "com.liferay.portal.service.permission",
         "com.liferay.portal.servlet",
-        "com.liferay.portal.servlet.filters.authverifier",
```

Comment:
Btw this change has no relationship with code upgrade tool, I just reviewed the code and found it.